frontend/ChapterList.vue

```diff
@@ -865,19 +888,44 @@ export default Vue.extend({
       }
       return c;
     },
-    colorQuarterlyGoal(text: string) {
+    colorLastAction(text: string) {
+      const GREEN = 'is-success';
+      const YELLOW = 'is-warning';
+      const RED = 'is-danger';
+      const GRAY = 'is-grey';
+      const BLACK = 'is-black';
```

Comment:
nit: could move these to module scope. i would prob do smth like this fwiw:
```ts
const Colors = {
  GREEN: "is-success",
  YELLOW: "is-warning",
  ...
}
```

Reply:
done